docs/vizier/vizier.rst

```diff
@@ -40,55 +40,96 @@ the :meth:`~astroquery.vizier.VizierClass.find_catalogs` tool can be used:
     J/AJ/157/217 : Transit times of five hot Jupiter WASP exoplanets (Bouma+, 2019)
     J/A+A/635/A122 : CoRoT-30b and CoRoT-31b radial velocity curves (Border+, 2020)
 
-From this result, you could either get any of these as a complete catalog or
-query them for individual objects or regions.
+From this list, you can either get any catalog completely (all the lines) or make
+sub-selections with :meth:`~astroquery.vizier.VizierClass.query_region`, 
+or with additional criteria.
 
 Get a whole catalog
 -------------------
 
+From the result of the precedent example, let's select ``J/ApJ/788/39``.
+We call :meth:`~astroquery.vizier.VizierClass.get_catalogs`. Let's fix the number of
+returned rows to 1 for now, to minimize the amount of data we download and speed up exploration, while we inspect the result:
```

Comment:
my previous comment made this go over 80 characters on one line.... it might be good to put this back through your own editor and reformat the whitespace

Reply:
Ah yes, the commit is amended